setup.py

```diff
@@ -1,29 +1,38 @@
 #!/usr/bin/env python
-try:
-    from setuptools import setup
```

Comment:
I think you need to keep this try/except for py2 compatibility

Reply:
it worked for me locally :D 

```bash
(venv2) ➜  venv2/bin/python --version
Python 2.7.17
(venv2) ➜  venv2/bin/python setup.py install
/.../Versions/2.7/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'install_requires'
  warnings.warn(msg)
running install
running build
running build_py
running build_ext
skipping 'credis/base.c' Cython extension (up-to-date)
skipping 'credis/geventpool.c' Cython extension (up-to-date)
running install_lib
running install_egg_info
Removing /..../venv2/lib/python2.7/site-packages/credis-2.0.0-py2.7.egg-info
Writing /..../venv2/lib/python2.7/site-packages/credis-2.0.0-py2.7.egg-info
(venv2) ➜  echo $?
0
```